### ISSUE TITLE
parmap doesn't work super well?!

### ISSUE THREAD

**Issue by UnixJunkie** [igarnier/k-means#1]
you can try parany (in opam) as a simple alternative.
But parmap is supposed to work very well, in many use cases!

**Comment by UnixJunkie** [comment]
try ~chunksize:1 with parmap: it turns ON the load balancing (which is OFF by default).
If you don't like this default (I don't), please tell it here: https://github.com/rdicosmo/parmap/issues/47

**Comment by UnixJunkie** [comment]
;)

**Comment by igarnier** [comment]
Yeah it worked very well on my previous use-case (heavy numerical code) but when trying to parallelize the multi-start routine, all processes were running on the same cpu. I didn't have time to investigate further, perhaps it comes from my configuration. I'll try what you suggest
